frontend/assets/javascripts/src/modules/identityMenu/identityPopup.js

```diff
@@ -24,22 +24,22 @@ define([
     }
 
     function disableLink() {
-        bean.on(IDENTITY_MENU_CTA_ELEM, 'click', function(e) {
+        IDENTITY_MENU_CTA_ELEM.on('click', function(e) {
```

Comment:
If this is the only place `bean` is used in this file, could we just do `$(IDENTITY_MENU_CTA_ELEM)` here, and leave the native browser methods intact everywhere else?